3p/ampcontext.js

```diff
@@ -100,7 +100,7 @@ export class AbstractAmpContext {
     /** @protected {!IframeMessagingClient} */
     this.client_ = new IframeMessagingClient(win);
     this.client_.setHostWindow(this.getHostWindow_());
-    this.client_.setSentinel(dev().assertString(this.sentinel));
```

Comment:
Ok, changed back to dev() here, which now leaves this file as unchanged in this PR. Also made the same change in 3p/iframe-transport-client.js which is new to this PR, but it's the one from #10596 that started the whole conversation.